contents/blog/dev-marketing-for-startups.md

```diff
@@ -0,0 +1,98 @@
+
+In your product signup flow, include an optional free text box asking people where they first heard about you. About 10% of signups usually fill this in. Read the data and report on it every week. It’s manual but vitally important info.
+
+### Post-product-market fit
+
+**Paid ads are not all bad** 
+
+Paid ads can be a useful tool for learning quickly if you are building a consumer app, but _not_ for devtools (see ‘attribution mirage’ above), which is why I’ve put them in post-PMF. It is very easy to spend tons of money on shit, especially in Google Ads. NEVER USE THE DEFAULT SETTINGS on Google, LinkedIn, and Twitter especially. Hire a cheap agency. We aim to be 7/10 at paid ads, not amazing. Focus that energy on your content instead. 
+
+Google is good for conversion, while LinkedIn ($$$), Twitter ($$), Reddit ($), and Quora ($) are good for general awareness. Avoid Product Hunt if you sell something that costs > $10/month (you’ll get loads of signups, but they hate spending money). Carbon Ads is very expensive and was a waste of time for us. 
+
+**It’s ok to waste money on sponsorships** 
+
+They are generally quite expensive, but unfortunately you need to spend money to learn here. Newsletters are the best place to start - they can be highly effective once you find the right one. Ask your engineers and users what newsletters they like, then sponsor those. Do bursts of activity - e.g. 3 months on, 3 months off. Switch up the copy so you don't exhaust the audience.
+
+YouTube is hard and a much more expensive way to reach developers. Most dev-focused content is for people learning to code, so beware.
+
+### Two things we haven’t figured out yet
+
+**Events are 10x more work than you think** 
+
+Don’t host your own - you do not have the budget to do this well. Do attend others though, if you have someone on your team who loves doing them and is an engineer. Sending marketing people to events only is not leveraged enough. For now, we give our entire team budget to attend events on PostHog's behalf and leave it at that. Sponsoring events is disproportionately expensive - and if your name appears on the same level as Google's, you are definitely wasting your money. 
+
+**It's ok for other commpanies to be much better than you at social media** 
+
+We’re seeing promising results on Twitter, but it’s too early to share concrete feedback. One thing we have learned is to not try multiple channels at the same time - you just feel like you’ve done an ‘ok’ job at everything, and you don’t truly learn what works. We’ve found posting from personal accounts 10x more effective than posting from the company account. Bear in mind that the companies that do an excellent job at social media put _a lot_ of work into making it look effortless - don't underestimate the effort required. 
```

Comment:
Aren't we talking about multiple channels at the same time in this guide? Maybe clarify that it is multiple social channels (such and Twitter, LinkedIn, and TikTok).

Reply:
Good point, will tweak slightly